python-package/lightgbm/plotting.py

```diff
@@ -453,6 +453,7 @@ def _to_graphviz(
     feature_names: Union[List[str], None],
     precision: Optional[int],
     orientation: str,
+    max_category_values: Optional[int],
```

Comment:
Can you please move this to the end of the signature (before kwargs)?

Reply:
done